### PR TITLE
Create rexel_gb.py

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

**Comment by peternewman** [comment]
After #4708 .

File: locations/spiders/rexel_gb.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from locations.storefinders.rexel import RexelSpider
+
+
+class RexelGBSpider(RexelSpider):
+    name = "rexel_gb"
+    item_attributes = {"brand": "Rexel", "brand_wikidata": "Q"}
```
Comment (Cj-Malone) [inline locations/spiders/rexel_gb.py:6]:
```suggestion
    item_attributes = {"brand": "Rexel", "brand_wikidata": "Q962489"}
```

Comment (peternewman) [reply locations/spiders/rexel_gb.py:6]: I'd seen the QID but wasn't sure if it was French only.

I just went to do this, but you've beat me to it! :
https://github.com/osmlab/name-suggestion-index/commit/4d07c92457d0c7f2c1d1b6048228a8d6f37a90d8

**Comment by peternewman** [comment]
Superceded by #4713 for now as I merged in the wrong direction :facepalm:

**Comment by Cj-Malone** [comment]
I'm going to close this for now, if needed it can be re opened.

@peternewman are you doing all this development in GitHub and the actions? Selfishly can I ask you to try and set up a local environment, the docs changed lately we need a new person to follow them to see if they make sense.

**Pull request closed without merging**